starport_template/pubspec.yaml

```diff
@@ -24,6 +24,7 @@ dependencies:
   intl: 0.17.0
   alan: 0.40.2+1
   shared_preferences: 2.0.12
+  flutter_masked_text2: ^0.9.1
```

Comment:
![CleanShot 2022-03-31 at 13 41 08@2x](https://user-images.githubusercontent.com/1711474/161057305-3bbe0f33-3b59-4106-9b07-fa1d3a326eba.png)
 this is exactly the reason why we refrain from putting too many libraries into the template, let's not impose those dependencies on end users and keep the template as simple as possible.
 
I believe masked text is not a core feature of the template and can be  easly omitted

Reply:
also, this looks like its not used anywhere

Reply:
Yeah i was going to remove it